src/views/createCommunity/index.tsx

```diff
@@ -368,10 +369,10 @@ function CreateCommunityScreen() {
                 setIsEmailValid(false);
             }
 
-            const _isEnabledGPS = gpsLocation !== undefined;
-            if (!_isEnabledGPS) {
-                setIsEnabledGPS(false);
-            }
+            // const _isEnabledGPS = gpsLocation !== undefined;
```

Comment:
commented?

Reply:
Good eye. Forgot to undo this ... finished this too late ... Fixed.